clang/test/AST/Interp/intap.cpp

```diff
@@ -9,8 +9,7 @@ using MaxBitInt = _BitInt(128);
 
 constexpr _BitInt(2) A = 0;
 constexpr _BitInt(2) B = A + 1;
-constexpr _BitInt(2) C = B + 1; // expected-warning {{from 2 to -2}} \
-                                // ref-warning {{from 2 to -2}}
+constexpr _BitInt(2) C = B + 1;
```

Comment:
Losing this diagnostic is definitely incorrect, this is one we should have kept.